typedoc.json

```diff
@@ -8,7 +8,7 @@
     "dom",
     "esnext"
   ],
-  "exclude": ["**/*.spec.ts", "examples/**", "**/node_modules/**"],
+  "exclude": ["**/*.spec.ts", "**/spec.util.ts", "examples/**", "**/node_modules/**"],
```

Comment:
smart!

Reply:
needed to get those deps over to devdeps